src/main/java/com/github/manolo8/darkbot/core/manager/RepairManager.java

```diff
@@ -80,18 +81,45 @@ public boolean setBeforeReviveTime() {
         return System.currentTimeMillis() - beforeReviveTime < (main.config.GENERAL.SAFETY.WAIT_BEFORE_REVIVE * 1000L);
     }
 
+    private boolean instantRepaired = true;
```

Comment:
place all the state of the class at the top whenever possible

also, i'd consider swapping this one around, so it's "shouldInstaRepair = false", set to true once you die (if config allows), and re-set back to false after setting

right now, you're setting instantRepaired to true to "disable" instnatrepairing, even if you *didn't* really instant repair, so what the variable represents and its name doesn't quite align